Formula/git-when-merged.rb

```diff
@@ -1,21 +1,21 @@
 class GitWhenMerged < Formula
-  include Language::Python::Shebang
-
   desc "Find where a commit was merged in git"
   homepage "https://github.com/mhagger/git-when-merged"
   url "https://github.com/mhagger/git-when-merged/archive/v1.2.1.tar.gz"
   sha256 "46ba5076981862ac2ad0fa0a94b9a5401ef6b5c5b0506c6e306b76e5798e1f58"
   license "GPL-2.0-only"
+  head "https://github.com/mhagger/git-when-merged.git", branch: "master"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "cef2e377082bfa5b6457f6b6d19066fd7a74312860533b85fb62d6e16c0986b7"
   end
 
-  depends_on "python@3.10"
+  depends_on "python@3.10" => :test
+  uses_from_macos "python"
 
   def install
-    rewrite_shebang detected_python_shebang, "bin/git-when-merged"
-    bin.install "bin/git-when-merged"
+    bin.install "src/git_when_merged.py" => "git-when-merged"
+    inreplace bin/"git-when-merged", %r{^#!/usr/bin/env python$}, "#!/usr/bin/env python3"
```

Comment:
```suggestion
    inreplace bin/"git-when-merged", %r{^#!/usr/bin/env python$}, "#!/usr/bin/env python3" if OS.mac? && MacOS.version < :catalina
```

I think? Technically should modify the test too but that's probably not important.